semantic/src/site/collections/table.overrides

```diff
@@ -190,6 +190,17 @@
     }
   }
 
+  #applications-list {
+    .ui.table tr td:nth-last-child(2).has-mobile-action-cell {
+      border-bottom-left-radius: 10px;
+      border-bottom-right-radius: 10px;
+      padding-bottom: 10px !important;
+    }
+    .ui.table tr td:last-child:not(.has-mobile-action-cell) {
+      padding-bottom: 10px !important;
+    }
+  }
```

Comment:
So the main problem was that when the "Action" cell was removed from the normal flow (using `position: absolute`), it is still considered the "last child" of the container. So then the rounded bottom would be missing from the the cell. So I had to do some messy business to determine if there is an action cell or not, and style either the last child or the 2nd-to-last accordingly.